main/guides/UIComponentLibrary/UIComponentsstart.md

```diff
@@ -0,0 +1,109 @@
+
+You'll find a collection of ready-to-use or pre-coded user interface elements that help accelerate your Agoric Dapp-building. These pre-designed elements include wallet connect buttons, chain selector, node selector, amount input and wallet provisioning.
+Utilization of these building blocks ensures best practices in design and accessibility.
+
+## Installation and Setup
+
+See [Introduction to Agoric UI Kit](https://a0a31cba.documentation-7tp.pages.dev/guides/getting-started/how-to-use-ui-kit#introduction-to-agoric-ui-kit) for installation and setup how-to.
```

Comment:
This probably shouldn't link to a CI rendering.

Reply:
quite:

> use relative links instead of absolute ones for any links to files or folders in the Documentation repo
  -- [Writing Links in CONTRIBUTING]